### PR TITLE
DEVX-2301: update repo to conform to install.yml changes

### PR DESCRIPTION
**Pull request by AmyJZhao**
## Description

install.yml configuration was updated to rename `services` to `apis`
https://wiki.corp.adobe.com/pages/viewpage.action?spaceKey=AdobeCloudPlatform&title=App+Builder+Template+Install+Configuration

## How Has This Been Tested?

Updated tests
`npm run test`

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
- [X] I have added tests to cover my changes.
- [X] All new and existing tests passed.

### CODE REVIEW

File: registry.schema.json [hunk 1]
```diff
@@ -66,7 +66,7 @@
           ]
         }
       },
-      "services": {
+      "apis": {
         "type": "array",
         "description": "A list of Adobe services required by the template",
```
Comment (slitviachenko) [inline registry.schema.json:71]: Let's rename to
```suggestion
        "description": "A list of Adobe APIs required by the template",
```

Comment (slitviachenko) [inline registry.schema.json:71]: Let's rename to
```suggestion
        "description": "A list of Adobe APIs required by the template",
```

File: src/add-to-registry.js [hunk 2]
```diff
@@ -22,7 +22,6 @@ const { isInRegistry, addToRegistry, getFromRegistry, updateInRegistry, TEMPLATE
             "description": npmPackageMetadata.description,
             "latestVersion": npmPackageMetadata.version,
             "publishDate": (new Date(Date.now())).toISOString(),
-            "services": npmPackageMetadata.services,
```
Comment (slitviachenko) [inline src/add-to-registry.js:25]: We need to change it to "categories" as categories are mandatory in `install.yml`.
And an `if` block below
```
        if (npmPackageMetadata.categories) {
            templateData['categories'] = npmPackageMetadata.categories;
        }
```
change to
```
        if (npmPackageMetadata.apis) {
            templateData['apis'] = npmPackageMetadata.apis;
        }
```

We will have
```
        let templateData = {
            "author": npmPackageMetadata.author,
            "name": npmPackageMetadata.name,
            "description": npmPackageMetadata.description,
            "latestVersion": npmPackageMetadata.version,
            "publishDate": (new Date(Date.now())).toISOString(),
            "categories": npmPackageMetadata.categories,
            "adobeRecommended": adobeRecommended,
            "keywords": npmPackageMetadata.keywords,
            "status": TEMPLATE_STATUS_APPROVED,
            "links": {
                "npm": npmUrl,
                "github": gitHubUrl
            }
        }
        if (npmPackageMetadata.extension) {
            templateData['extension'] = npmPackageMetadata.extension;
        }
        if (npmPackageMetadata.apis) {
            templateData['apis'] = npmPackageMetadata.apis;
        }
```

File: src/add-to-registry.js [hunk 3]
```diff
@@ -31,6 +30,10 @@ const { isInRegistry, addToRegistry, getFromRegistry, updateInRegistry, TEMPLATE
                 "github": gitHubUrl
             }
         }
+
+        if(npmPackageMetadata.apis) {
```
Comment (slitviachenko) [inline src/add-to-registry.js:34]: You don't need this block once you apply my suggestion above.

File: src/update-template.js [hunk 4]
```diff
@@ -19,7 +19,6 @@ const { getFromRegistry, updateInRegistry } = require('./registry');
             "name": npmPackageMetadata.name,
             "description": npmPackageMetadata.description,
             "latestVersion": npmPackageMetadata.version,
-            "services": npmPackageMetadata.services,
```
Comment (slitviachenko) [inline src/update-template.js:22]: The same as for `src/add-to-registry.js` we need to change "services" to "categories" as categories are mandatory in `install.yml`. And update [this block](https://github.com/adobe/aio-template-submission/blob/c7d7b9d955ee46f870361d9278ab0eef3b5b8646/src/update-template.js#L33-L35) to
```
        if (npmPackageMetadata.apis) {
            templateData['apis'] = npmPackageMetadata.apis;
        }
```

We will have
```
        let templateData = {
            "author": npmPackageMetadata.author,
            "name": npmPackageMetadata.name,
            "description": npmPackageMetadata.description,
            "latestVersion": npmPackageMetadata.version,
            "categories": npmPackageMetadata.categories,
            "adobeRecommended": adobeRecommended,
            "keywords": npmPackageMetadata.keywords,
            "links": {
                "npm": npmUrl,
                "github": gitHubUrl
            }
        };
        if (npmPackageMetadata.extension) {
            templateData['extension'] = npmPackageMetadata.extension;
        }
        if (npmPackageMetadata.apis) {
            templateData['apis'] = npmPackageMetadata.apis;
        }
```

**Review by slitviachenko — CHANGES_REQUESTED** [review]
@AmyJZhao , I left some comments, please take a look at them.
Please also add a support of _optional_ `runtime` and `event` properties. We need to store them in Template Registry, and return via API.

**Review by slitviachenko — CHANGES_REQUESTED** [review]
Please update [@adobe/aio-lib-template-validation](https://github.com/adobe/aio-template-submission/blob/c7d7b9d955ee46f870361d9278ab0eef3b5b8646/package.json#L19) to a new [@adobe/aio-lib-template-validation:2.3.0](https://www.npmjs.com/package/@adobe/aio-lib-template-validation/v/2.3.0) version.

**Review by slitviachenko — CHANGES_REQUESTED** [review]
@AmyJZhao , Could you please address [this comment](https://github.com/adobe/aio-template-submission/pull/128#pullrequestreview-995619028) as well regarding _runtime_ and _event_.

File: registry.schema.json [hunk 5]
```diff
@@ -115,4 +157,4 @@
       "links"
     ]
   }
-}
+}
```
Comment (roribio) [inline registry.schema.json:160]: Missing new line.

**Review by roribio — COMMENTED** [review]
Same comment about `runtime` and `events` attributes. Otherwise, looks good 👍

**Review by slitviachenko — APPROVED** [review]

**Comment by AmyJZhao** [comment]
@roribio See Serhii's comment on the other PR. Shaz is planning on updating templates to use `runtime` and `event` in their configurations, so DEVX-2301 is for updating all of template registry ASAP.

**Pull request merged**